src/components/Dissolution/CompleteResolution.vue

```diff
@@ -497,19 +497,19 @@ export default class CompleteResolution extends Mixins(CommonMixin, DateMixin, E
   ]
 
   private get isResolutionDateValid (): boolean {
-    return this.$refs.resolutionDatePickerRef.isDateValid()
+    return this.$refs.resolutionDatePickerRef?.isDateValid()
```

Comment:
The "resolution page is invalid but all components are valid" bug did not reproduce for me locally. However, when doing hot-reload of code changes, I saw console errors about "unable to call isDateValid on undefined". So that's what this PR fixes. I don't know why the other component refs don't have a problem. This change can't hurt and might fix the issue in Dev.